cluster/gce/gci/configure.sh

```diff
@@ -296,7 +296,7 @@ function install-node-problem-detector {
   fi
 
   echo "Downloading ${npd_tar}."
-  local -r npd_release_path="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-https://storage.googleapis.com/kubernetes-release}"
+  local -r npd_release_path="${NODE_PROBLEM_DETECTOR_RELEASE_PATH:-"https://github.com/kubernetes/node-problem-detector/releases/download/${npd_version}"}"
   download-or-bust "${npd_hash}" "${npd_release_path}/node-problem-detector/${npd_tar}"
```

Comment:
`download-or-bust "${npd_hash}" "${npd_release_path}/${npd_tar}"`

Example: https://github.com/kubernetes/node-problem-detector/releases/download/v0.8.16/node-problem-detector-v0.8.16-linux_amd64.tar.gz

Reply:
That would break users currently setting the existing capital env?

Reply:
I guess we're forced to

Reply:
We are only making this change in master branch, right? Existing customers would not be impacted.

Reply:
I guess nothing is currently testing this aspect either since CI passed ..

Reply:
Nothing in the project is using this override https://cs.k8s.io/?q=NODE_PROBLEM_DETECTOR_RELEASE_PATH
And we don't support external users for these scripts (officially deprecated for years now), so it's OK to break.

Reply:
done.